### PR TITLE
[WIP] Create supported-os-matrix.md

### PR DESCRIPTION
**Pull request by blackdwarf**
Adding a document that outlines goals for supported operating systems for the CLI

skip ci please

/cc @richlander @piotrpMSFT @Petermarcu

---
This change is [Reviewable](https://reviewable.io/reviews/dotnet/cli/1923)

### CODE REVIEW

File: Documentation/supported-os-matrix.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+- Debian 8.2
+- Ubuntu 14.04 LTS
+- CentOS 7.1
+- Red Hat Enterprise Linux 7.2
+
+## OS X (x64 only)
+The following versions of OS X are supported:
+
+- OS X 10.11+ (El Capitan)
```
Comment (enricosada) [inline Documentation/supported-os-matrix.md]: no `osx-10.10` ?

**Comment by blackdwarf** [comment]
Ping...

**Comment by blackdwarf** [comment]
@leecow can you take a look at this and "bless" it so I can merge?

**Comment by leecow** [comment]
LGTM: Matches what is included in https://github.com/dotnet/core/blob/master/release-notes/1.0/Release-Notes-RC2.md

**Comment by blackdwarf** [comment]
Good, merging.

**Pull request merged**